rosdep/python.yaml

```diff
@@ -4135,6 +4135,18 @@ python-setuptools:
     xenial_python3: [python3-setuptools]
     yakkety: [python-setuptools]
     zesty: [python-setuptools]
+python-sexpdata:
+  debian:
+    '*':
+      pip:
+        packages: [sexpdata]
+    buster: [python-sexpdata]
+  ubuntu:
+    '*':
+      pip:
+        packages: [sexpdata]
+    disco: [python-sexpdata]
+    eoan: [python-sexpdata]
```

Comment:
Same goes here; we should make the package the '*' (so we will get it automatically for future Ubuntu releases), and just call out the still supported ones with pip (in this case, it's just 'xenial' and 'bionic').